adapters/src/main/java/io/realm/RealmRecyclerViewAdapter.java

```diff
@@ -85,12 +79,7 @@ public RealmRecyclerViewAdapter(@Nullable OrderedRealmCollection<T> data, boolea
         // Right now don't use generics, since we need maintain two different
         // types of listeners until RealmList is properly supported.
         // See https://github.com/realm/realm-java/issues/989
```

Comment:
is the above assumption still valid? https://github.com/realm/realm-java/issues/989  is closed now?